MediaBrowser.Api/Playback/BaseStreamingService.cs

```diff
@@ -311,7 +311,7 @@ private async Task AcquireResources(StreamState state, CancellationTokenSource c
             {
                 StartThrottler(state, transcodingJob);
             }
-            Logger.LogDebug("StartFfMpeg() finished successfully");
+            Logger.LogError("StartFfMpeg() finished successfully");
```

Comment:
```suggestion
            Logger.LogInformation("StartFfMpeg() finished successfully");
```

Reply:
I don’t think this line should be in info, maybe make it more user friendly.

Reply:
Why not?

Reply:
`StartFfMpeg()` the function name

Reply:
I can see how it looks in the logs now as an end user and see if just saying FFmpeg starts/stops works well